index.bs

```diff
@@ -195,7 +195,8 @@ Non-Functional Requirements {#requirements-non-functional}
 1.  It should be possible to implement an OSP agent using modest
     hardware requirements, similar to what is found in a low end smartphone,
     smart TV or streaming device. See the [Device
-    Specifications](device_specs.md) document for agent hardware specifications.
+    Specifications](https://github.com/w3c/openscreenprotocol/blob/master/device_specs.md)
```

Comment:
Maybe these links should point to the gh-pages branch since we are serving the spec document from that branch?

I don't feel strongly since the two branches should always be kept in sync by the workflow.

Reply:
I would keep the `gh-pages` name as a "hidden" one: no need to expose the name of the branch that gets published as GitHub Pages, to avoid breaking links if we change the name. Actually, that applies to `master` as well, which we'll soon need to rename to `main` (we're progressively transitioning all W3C repositories to use `main` as the default branch instead of `master`).

I thought something had to be done for GitHub to publish Markdown files as HTML but I was wrong. It's all being done by default, you just have to replace the `.md` extension with `.html`. I pushed an update that targets the HTML pages under `w3c.github.io` (i.e. [capabilities.html](https://w3c.github.io/openscreenprotocol/capabilities.html) and [device_specs.html](https://w3c.github.io/openscreenprotocol/device_specs.html)). We can always choose a different theme and adjust the header and footer on these pages later on if we feel that's needed.